dvc/command/cache.py

```diff
@@ -7,6 +7,9 @@
 
 class CmdCacheDir(CmdConfig):
     def run(self):
+        if self.args.value is None and not self.args.unset:
+            logger.info(self.config["cache"]["dir"])
```

Comment:
Unlike https://github.com/iterative/dvc/blob/738dfd49cf403412b018677e62b0b68e85c80886/dvc/command/config.py#L19 , we are not taking `level` into account :slightly_frowning_face:

Reply:
My bad. I should have looked at logging.
Thanks for correcting.
Will take a pull and test it once.